provider/cmd/pulumi-resource-google-cloud-static-website/website.ts

```diff
@@ -80,7 +80,7 @@ export class Website extends pulumi.ComponentResource {
             }, { parent: this });
         });
 
-        this.originURL = pulumi.interpolate`https://storage.googleapis.com/${bucket.name}/index.html`;
+        this.originURL = pulumi.interpolate`https://storage.googleapis.com/${bucket.name}/${this.indexDocument || "index.html"}`;
```

Comment:
Is this necessary, given the default on line 45?

Reply:
ahh, yes correct. it is already being set. thanks.